hack/lib/cleanup.sh

```diff
@@ -90,14 +90,15 @@ function os::cleanup::tmpdir() {
         for target in $( ${USE_SUDO:+sudo} findmnt --output TARGET --list ); do
             if [[ "${target}" == "${BASETMPDIR}"* ]]; then
                 ${USE_SUDO:+sudo} umount "${target}"
+                ${USE_SUDO:+sudo} rm -rf "${target}"
             fi
         done
+    else
+        # in case where there's no findmnt remove all directories
```

Comment:
This is not the correct logic. We want to delete the directories _always_ but we only try to unmount stuff best-effort. We have never run into the case where the user did not have `findmnt` but did have mounts -- usually they are on Mac without `findmnt` and the mounts are in the docker VM.